### ISSUE TITLE
HTTPS Frontend don't work

### ISSUE THREAD

**Issue by Orabee** [Project-OSRM/osrm-frontend-v2#136]
Link : https://map.project-osrm.org

Regards,
Vincent

**Comment by TheMarex** [comment]
In what browser? Seems to work in Firefox.

**Comment by danpat** [comment]
Chrome is giving me back the certificate for "www.github.com"
<img width="814" alt="screen shot 2015-12-16 at 7 52 26 am" src="https://cloud.githubusercontent.com/assets/1892250/11845710/fde62e7a-a3c9-11e5-8c5e-cc8a291b45d0.png">

**Comment by Orabee** [comment]
Try with Chrome, Safari and Firefox on mac. Need to validate certificate each time.

**Comment by danpat** [comment]
@Orabee No, I was pointing out that the incorrect HTTPS certificate is being served. 

The underlying problem is Github.io not supporting custom HTTPS certificates on custom domains:

https://github.com/isaacs/github/issues/156

**Comment by Orabee** [comment]
Or use Let's encrypt (https://letsencrypt.org/) ? Free SSL Certificates...

**Comment by danpat** [comment]
@Orabee the problem isn't that we don't have a certificate (we do), it's that where we host these pages (github.io) doesn't support SSL certificates for custom domains, as mentioned https://github.com/Project-OSRM/osrm-frontend-v2/issues/136#issuecomment-165255689

To fix this, we would need to host `map.project-osrm.org` somewhere else.
